keyboards/hardlight/keymaps/default/keymap.c

```diff
@@ -0,0 +1,132 @@
+  KC_ESC,  KC_A,    KC_O,    KC_E,    KC_U,    KC_I,    KC_D,    KC_H,    KC_T,    KC_N,    KC_S,    KC_SLSH, SET,     KC_KP_4, KC_KP_5,   KC_KP_6,
+  KC_LSFT, KC_SCLN, KC_Q,    KC_J,    KC_K,    KC_X,    KC_B,    KC_M,    KC_W,    KC_V,    KC_Z,    KC_UP,   KC_ENT,  KC_KP_1, KC_KP_2,   KC_KP_3,
+  KC_LCTL, KC_PAUS, KC_LGUI, KC_LALT, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_LEAD, KC_RSFT, KC_LEFT, KC_DOWN, KC_RGHT, KC_KP_0, KC_KP_DOT, KC_PENT
+  ),
+
+ /* SETTINGS
+ * .-----------------------------------------------------------------------------------------------------------------------------------------------.
+ * |        |        |        |        |        |        |        |        |        |        |        |        |        | RGBPWR |  Mode  |  Test  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | Reset! |        |        |        |        |        |        |        |        |        |        |        |        |  HUE+  |  SAT+  |  VAL+  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |        |        |        |        |        |        |        |        |        |        |        |        | NumLok |  HUE-  |  SAT-  |  VAL-  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |        |        |        |        |        |        |        |        |        | CapLok |        | ScrLok |        |  Plain | RSwirl | Larsen |
+ * '-----------------------------------------------------------------------------------------------------------------------------------------------'
+ */
+
+  [_SET] = LAYOUT_ortho_4x16(
+  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   RGB_TOG, RGB_MOD,  RGB_M_T,
+  RESET,     _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   RGB_HUI, RGB_SAI,  RGB_VAI,
+  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_NLCK,   RGB_HUD, RGB_SAD,  RGB_VAD, 
+  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_CAPS,   _______,   KC_SLCK,   _______,   RGB_M_P, RGB_M_SW, RGB_M_K
+  ),
+
+};
+
+LEADER_EXTERNS();
+
+void matrix_scan_user(void) {
+  LEADER_DICTIONARY() {
+    leading = false;
+    leader_end();
+
+    SEQ_FOUR_KEYS(KC_M, KC_A, KC_I, KC_L) {
+      SEND_STRING("https://mail.google.com/"SS_TAP(X_ENTER));
+    }
+    SEQ_ONE_KEY(KC_RSFT) {
+      register_code(KC_LGUI);
+      register_code(KC_L);
+      unregister_code(KC_L);
+      unregister_code(KC_LGUI);
```

Comment:
```suggestion
      tap_code16(G(KC_L));
```

What you have here is fine, but can be done more succinctly with `tap_code`.